beacon_node/eth1/src/service.rs

```diff
@@ -613,7 +631,37 @@ impl Service {
                 cache_4.block_cache.read().len() as i64,
             );
 
-            Ok(BlockCacheUpdateOutcome::Success {
+            let block_cache = service_1.inner.block_cache.read();
+            let latest_block_mins = block_cache
+                .earliest_block_timestamp()
```

Comment:
Should this be `latest_block_timestamp`?

Reply:
Great pickup, thanks!